src/styles.ts

```diff
@@ -57,29 +99,45 @@ export class StyleCompiler extends EventEmitter {
       }));
   }
 
-  private postprocess(processed: any) {
+  private postprocess(output: string, processed: any) {
     return new Promise((resolve, reject) => {
-      postcss(this.config.postcss.map(([pluginName, pluginConfig]) =>
-        require(pluginName)(pluginConfig)))
-        .process(processed.css, {map: {inline: false}, to: this.config.output})
+      postcss((this.config.postcss || []).map(([pluginName, pluginConfig]) => {
+        const plugin = require(pluginName);
+        if (pluginConfig) {
```

Comment:
I get your point.
But the issue with your suggestion is that there is at least one postcss plugin that is not configurable and does not export a function. See https://github.com/luisrudge/postcss-flexbugs-fixes#usage for example.
My solution now is: check if the export is a function with `_.isFunction()` and call it only in that case.